rules/linux/credential_access_potential_linux_ssh_bruteforce_external.toml

```diff
@@ -17,6 +17,7 @@ from = "now-9m"
 index = ["logs-system.auth-*"]
 language = "eql"
 license = "Elastic License v2"
+max_signals = 1
```

Comment:
what does this do?

Reply:
This makes sure that for example if the rule runs over a dataset of 5 minutes, and would trigger 100 alerts in that timespan, that only 1 alert will be generated. This will reduce noise greatly for these rules, as a brute force attack may generate 100+ alerts in less than 5 minutes.